doc/userguide/install.rst

```diff
@@ -94,23 +71,19 @@ Ubuntu/Debian
 
 Minimal::
 
-    # Installed Rust and cargo as indicated above
-    sudo apt-get install build-essential git libjansson-dev libpcap-dev \
-                    libpcre2-dev libtool libyaml-dev make pkg-config zlib1g-dev
-    # On most distros installing cbindgen with package manager should be enough
-    sudo apt-get install cbindgen # alternative: cargo install --force cbindgen
+    sudo apt-get install build-essential cargo cbindgen git libjansson-dev \
+                    libpcap-dev libpcre2-dev libtool libyaml-dev make \
+                    pkg-config rustc zlib1g-dev
```

Comment:
I noticed that this doesn't list either `gcc` or `clang`, which are indicated as required in a paragraph that was removed. Is that alright?

Reply:
I think its a bit of an Ubuntu'ism to just install `build-essential` to get all that stuff rather than breaking it out.